src/lib/BgColorClass.js

```diff
@@ -0,0 +1,7 @@
+const BgColorClass = {
```

Comment:
Just a thought , I think it is a common convention to use camelCase with js objects and also matches with rest of codebase

Reply:
changed.